test/recipe-test.js

```diff
@@ -289,3 +290,39 @@ describe('filterRecipes', () => {
     expect(filteredRecipes).to.deep.equal(expectedRecipes);
   })
 });
+
+describe('search recipes', () => {
```

Comment:
Love all these tests!!! Maybe we should have one for what happens when a user clicks the search button when no search term is entered in the search box?  The current behavior right now is that nothing happens.

Maybe connectedly, there is also no way to "reset" the search back to the initial state with all of the recipes?

Reply:
Great points, added a conditional and tests for this -> 

```
 it('should return an empty array for non matches', () => {
    let filteredRecipes = searchRecipes(sampleRecipeData, sampleIngredientsData, 'no match string');
    let otherFilteredRecipes = searchRecipes(sampleRecipeData, sampleIngredientsData, '')
    assert.deepEqual(filteredRecipes, []);
    assert.deepEqual(otherFilteredRecipes, [])
  });

  it('should return all recipes if only a space is entered', () => {
    let allRecipes = searchRecipes(sampleRecipeData, sampleIngredientsData, ' ');
    assert.deepEqual(allRecipes, sampleRecipeData)
  }) 
```

Made a new card for resetting the search